internal/projection/enriched_event_projection_test.go

```diff
@@ -36,23 +37,30 @@ var _ = Describe("Process message", func() {
 		ctrl = gomock.NewController(GinkgoT())
 		mockSnapshotRepo = redis_repo.NewMockSnapshotRepositoryInterface(ctrl)
 		mockEnrichedEventRepo = opensearch_repo.NewMockEnrichedEventRepositoryInterface(ctrl)
+		ackChannel = make(chan kafka.Message, 100)
```

Comment:
where is the `ackChannel` consumed? Once the channel will contain 100 items, it will become blocking, no? (my go-fu is rusty :sweat_smile: )

Reply:
> Once the channel will contain 100 items, it will become blocking, no?

Yes, exactly. This is the test file, and within the specs that are supposed to send an ack message, the channel will be consumed in order to make some assertion. AfterEach spec the channel will be closed (even if it gets overridden each spec in any case).

Reply:
arf, sorry didn't notice it was the test :man_facepalming: , because I was wonfdering where this channel is created and consumed

Reply:
this channel is created in the[ app entrypoint ](https://github.com/openshift-assisted/assisted-events-stream/blob/master/cmd/consumer/main.go#L21C1-L22C1) and consumed in the [kafka reader](https://github.com/openshift-assisted/assisted-events-stream/blob/master/pkg/stream/reader.go#L136C1-L146C3)